system/jlib/jutil.cpp

```diff
@@ -3030,6 +3030,70 @@ int getEnum(const char *v, const EnumMapping *map, int defval)
 
 //---------------------------------------------------------------------------------------------------------------------
 
+extern jlib_decl offset_t friendlyStringToSize(const char *in)
+{
+    char *tail;
+    offset_t result = strtoull(in, &tail, 10);
+    offset_t scale = 1;
+    if (*tail)
+    {
+        if (tail[1] == '\0')
```

Comment:
Does there have to be a suffix ?
Is the suffix always > 1 char ?

Reply:
if *tail is not zero, then tail[1] is ok to access.